protocol/src/main/java/dev/keva/protocol/resp/Command.java

```diff
@@ -180,4 +185,15 @@ public void toArguments(Object[] arguments, Class<?>[] types, ChannelHandlerCont
     public void write(ByteBuf os) throws IOException {
         writeDirect(os, name, object1, object2, object3, objects);
     }
+
+    public byte[] getOriginal() throws IOException {
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
```

Comment:
@haphananhtuan We consider not creating objects in protocol packages (to avoid creating garbage during parse/read conn)

Maybe can use this instead:

```java
byte[] concatBytes = ArrayUtils.addAll(one,two);
```

Reply:
Changed this to static object for reusability

Reply:
Thanks, LGTM @haphananhtuan